From fa874071de418d61abde3da422ab6bf8cc30e558 Mon Sep 17 00:00:00 2001
From: Patrice Chotard <patrice.chotard@foss.st.com>
Date: Thu, 28 Mar 2024 15:29:46 +0100
Subject: [PATCH 1100/1141] arm64: dts: st: Add button-user-2 on board
 stm32mp257f-ev1

Both button named USER1 and USER2 are available for Linux.

Change-Id: I88c8782bb084dda21b41f55da01240570e1fcf88
Signed-off-by: Patrice Chotard <patrice.chotard@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/369998
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 7 +++++++
 1 file changed, 7 insertions(+)

--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -72,6 +72,13 @@
 			status = "okay";
 		};
 
+		button-user-2 {
+			label = "User-2";
+			linux,code = <BTN_2>;
+			gpios = <&gpiog 8 GPIO_ACTIVE_HIGH>;
+			status = "okay";
+		};
+
 		button-wake-up {
 			label = "wake-up";
 			linux,code = <KEY_WAKEUP>;
